conf/messages.cy

```diff
@@ -887,6 +887,13 @@ benefits.childcareBenefitsAmount.error.noEntry.agent = Enter your client’s chi
 benefits.childcareBenefitsAmount.error.incorrectFormat.agent = Enter your client’s childcare benefit amount in the correct format
 benefits.childcareBenefitsAmount.error.overMaximum.agent = Your client’s childcare benefit must be less than £100,000,000,000
 
+benefits.educational-ServicesEmployment.title.individual = Did you get any benefits for educational services?
+benefits.educational-ServicesEmployment.title.agent = Did your client get any benefits for educational services?
+benefits.educational-ServicesEmployment.theseAre.individual = These are scholarships or school fees paid for by your employer.
```

Comment:
It seams that there is no test for the text value

Reply:
Same as above

Reply:
Same as above

Reply:
I didn't see them. Can we speak and you can show them to me